public/js/clients/chrome/events.js

```diff
@@ -46,12 +46,14 @@ async function paused(
     type: reason
   }, data);
 
+  pageAgent.configureOverlay(false, "Paused in debugger.html");
```

Comment:
I'm just calling both methods right now.

This needs some kind of protocol / feature detection.  If the protocol level was detecting the version and / or features when generating the methods our client code could be a bit simpler.  From the client we could do `if ("configureOverlay" in pageAgent)` to decide between the the two versions.  Will need a deeper dive.